pgxn/neon/neon.control

```diff
@@ -1,6 +1,6 @@
 # neon extension
 comment = 'cloud storage for PostgreSQL'
-default_version = '1.3'
```

Comment:
AFAIU it's updated in another PR https://github.com/neondatabase/neon/pull/8405

Reply:
It's now been merged